style.css

```diff
@@ -202,15 +211,12 @@ img {
     color: #F5F8FB;
     font-size: 52px;
     line-height: 62px;
-    font-family: "Bakbak One", Times New Roman, Georgia, serif;
+    font-family: var(--bakbak);
```

Comment:
As a suggestion, you can add to your variable a little bit more information. For example: `--font-secondary` or `--font-primary`. And after that you can change your font only in one place and it will be actual for all project.